app/cli/cmd/attestation_push.go

```diff
@@ -119,8 +127,11 @@ func newAttestationPushCmd() *cobra.Command {
 	cmd.Flags().StringVar(&signServerCAPath, "signserver-ca-path", "", "custom CA to be used for SignServer communications")
 	cmd.Flags().StringVar(&signServerAuthUser, "signserver-auth-user", "", "")
 	cmd.Flags().StringVar(&signServerAuthPass, "signserver-auth-pass", "", "")
+	cmd.Flags().BoolVar(&bypassPolicyCheck, "exception_bypass_policy_check", false, "do not fail this command on policy violations enforcement")
```

Comment:
One question, why the snake case with `_` rather than `-` as we have the other flags?

Reply:
hi, it's wrong, good catch

Reply:
Fixed, please review